pyproject.toml

```diff
@@ -120,9 +125,11 @@ overgeneral-exceptions = [
 ]
 
 [tool.pylint.TYPING]
-py-version = "3.8"
 runtime-typing = false
 
+[tool.pylint.CODE_STYLE]
+max-line-length-suggestions = 72
```

Comment:
Why is this?

Reply:
Currently, it's only for the `consider-using-assignment-expr` check. Although possible, it doesn't really make sense to use assignment expressions if the line gets too long and thus less readable. Form my testing `72` seems to be a good cutoff. Take a look at the example below, both options will work but the first is more readable IMO
```py
class X:
    def __init__(self):
        some_variable = self.some_really_long_method_call(argument1, argument2)
        if some_variable is not None:
            ...

        if (some_variable := self.some_really_long_method_call(argument1, argument2)) is not None:
            ...
```

Of course, it's still possible to use the second one. `pylint` just won't emit `consider-using-assignment-expr` for the first.